server/app.js

```diff
@@ -8,6 +8,7 @@ const dotenv = require("dotenv");
 
 ///////////////////////////////////////////// Configurations ////////////////////////////////////////
 const User = require('./models/user.model');  // register schema for model User
+const Activity = require('./models/activity.model')
```

Comment:
what is the purpose of having this constant? it does not seem to be used anywhere in this file. If no purpose, please get rid of it.